core/modules/wmgr/python/xrdMgr.py

```diff
@@ -120,6 +122,46 @@ def _restartService(sName):
 def _restartXrootd():
     _restartService('xrootd')
 
+
+def _workerId():
+    """ Fetch and return a unique identifier of the worker. """
+
+    _log.debug('fetching a unique identifier of the worker')
+
+    dbName = 'qservw_worker'
+    tblName = 'Id'
+
+    # check the db exists (listTables() does not fail on non-existing databases)
+    with Config.instance().dbEngine().begin() as dbConn:
+        if not utils.tableExists(dbConn, tblName, dbName):
+            raise ExceptionResponse(404, "TableMissing", "Table %s.%s does not exist" % (dbName, tblName))
+
+        query = "SELECT id FROM `%s`.`%s`" % (dbName, tblName)
```

Comment:
db/table names are fixed, just write query as "SELECT id FROM qservw_worker.Id". I don't use backticks unless it's necessary, would be easier to migrate to SQL Server without backticks :)

Reply:
I've removed the ticks. Though, I would still like to keep the separation for the sake of consistency with the code above that line.